aws/registers/register_company.tf

```diff
@@ -14,5 +14,6 @@ module "company" {
   dns_zone_id = "${module.core.dns_zone_id}"
   certificate_arn = "${var.elb_certificate_arn}"
 
+  enable_cdn = "${var.enable_cdn}"
```

Comment:
I don't completely understand what happens if `enable_cdn` is true, but `cloudfront_certificate_arn` is unset